app/src/main/java/app/gamenative/utils/ManifestRepository.kt

```
@@ -49,6 +57,12 @@ object ManifestRepository {
     }
 }
 
+    private fun readLocalManifest(context: Context): ManifestData? = try {
+        context.assets.open("manifest.json").bufferedReader().use { parseManifest(it.readText()) }
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)